x/auth/client/txbuilder/txbuilder.go

```diff
@@ -20,25 +22,38 @@ type TxBuilder struct {
 	SimulateGas   bool
 	ChainID       string
 	Memo          string
-	Fee           string
+	Fees          sdk.Coins
+}
+
+// NewTxBuilder returns a new initialized TxBuilder
+func NewTxBuilder(txEncoder sdk.TxEncoder, accNumber, seq, gas uint64, gasAdj float64, simulate bool, chainID, memo string, fees sdk.Coins) TxBuilder {
```

Comment:
Revisiting this again, I wonder if we can make all these fields private. Otherwise, having the `With*` methods here is senseless.

Reply:
Agree, I'll create the getter functions then